### PR TITLE
Updated the Installation instructions based on user feedback.  

### PR DESCRIPTION
**Pull request by twilson271828**
Also fixed a problem with the isis3VarInit.py script.

### CODE REVIEW

File: isis/scripts/isis3VarInit.py [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
 #Date  :  2018-10-05
 #Description:  This script sets ISISROOT/ISIS3DATA/ISIS3TESTDATA for the user and is executed
 #within the conda environment created for the ISIS3 installation.
 #The data directory and test directory are optional command line arguments.  If the user chooses
 #not to set them, they will both be placed created on the same level as the $ISISROOT directory
 #within the conda environment.
+#History:
+#   Author:  Tyler Wilson
+#   Date:    2018-11-01
+#   Description:  Removed a pair of lines which were causing output errors on Mac OS X and were note
```
Comment (kaitlyndlee) [inline isis/scripts/isis3VarInit.py]: Just a small typo, you mean not right?

File: isis/src/docsys/documents/InstallGuide/InstallGuide.xml [hunk 2]
```diff
@@ -34,43 +34,70 @@ you must <em>not</em> upgrade the ISIS Data Files!!!
...
 
-          python $CONDA_PREFIX/scripts/isis3VarInit.py
+          <strong>python $CONDA_PREFIX/scripts/isis3VarInit.py</strong>
 
          Executing this script with no arguments will result in $ISIS3DATA=$CONDA_PREFIX/data, 
          and $ISIS3TESTDATA=$CONDA_PREFIX/testdata.  The user can specify different directories
          for both of these optional values:
 
-         ./$CONDA_PREFIX/scripts/isis3VarInit --data-dir=[path to data directory]  --test-dir=[path to test data directory]
+         <strong>./$CONDA_PREFIX/scripts/isis3VarInit --data-dir=[path to data directory]  --test-dir=[path to test data directory]</strong>
```
Comment (kaitlyndlee) [inline isis/src/docsys/documents/InstallGuide/InstallGuide.xml:85]: Do we need to include the rsync command to get the ISIS3Data?

Comment (jessemapel) [reply isis/src/docsys/documents/InstallGuide/InstallGuide.xml:85]: That's later in the document. I could see a link here though.

File: isis/scripts/isis3VarInit.py (hunk 1 above)
Comment (twilson271828) [reply isis/scripts/isis3VarInit.py]: Fixed it.

File: isis/src/docsys/documents/InstallGuide/InstallGuide.xml (hunk 2 above)
Comment (twilson271828) [reply isis/src/docsys/documents/InstallGuide/InstallGuide.xml:85]: I placed an anchor link after this part to the area later in the document where we provide instructions for rsyncing the data.

Comment (jessemapel) [reply isis/src/docsys/documents/InstallGuide/InstallGuide.xml:85]: :+1:

Comment (kaitlyndlee) [reply isis/src/docsys/documents/InstallGuide/InstallGuide.xml:85]: Oh oops I did not look at the rest of the document, my bad. Looks good.

**Review by celbal — APPROVED** [review]

**Pull request merged**